io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-option:
+  - append-path: /usr/lib/sdk/rust-stable/bin
+    build-args:
+      - --share=network
+    env:
+      CARGO_HOME: /run/build/rust-flatpak/cargo
+modules:
+  - name: flutter
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://storage.googleapis.com/flutter_infra_release/releases/beta/linux/flutter_linux_2.6.0-5.2.pre-beta.tar.xz
```

Comment:
yes.

Reply:
because
```
% file bin/cache/dart-sdk/bin/dart
bin/cache/dart-sdk/bin/dart: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, stripped
```

doesn't look like it

Reply:
you are right, let me look into this.

Reply:
```
/app/lib/sdk/flutter/bin/internal/shared.sh: line 227: /app/lib/sdk/flutter/bin/cache/dart-sdk/bin/dart: cannot execute binary file: Exec format error
```

as I said, it doesn't work on `aarch64`.